FOR_DEVELOPERS.md

```diff
@@ -72,6 +72,34 @@ while True:
         break
 ```
 
+Download table data as a zip file. You can download all the table data in a data table in a single call. The following will download the entire F1 table data as a zip file to your current working directory:  
+
+```python
+import quandl
+data = quandl.export_table('MER/F1')
+```
+
+You can also specify where to download the zip file:  
+
+```python
+import quandl
+data = quandl.export_table('MER/F1', filename='/my/path/MER_F1_DB.zip')
+```
+
+Note that if you are downloading the whole table, it will take longer to generate the zip file.
+
+You can also specify what data you want to download with filters and parameters: 
+
+```python
+import quandl
+quandl.export_table('MER/F1', params={'compnumber': '39102', 'mapcode':'-5370','reporttype': 'A', 'qopts': {'columns': ['reportdate', 'amount']}})
+```
+
+After the download is finished, the filename of the downloaded zip file will be returned.
```

Comment:
Are you still returning the filename? I think you changed the method to only print the filename.

Reply:
updated